### PR TITLE
Consider the indirect owner-relationship between pod and workload

### PR DESCRIPTION
**Pull request by veophi**
Signed-off-by: mingzhou.swx <mingzhou.swx@alibaba-inc.com>

### Ⅰ. Describe what this PR does
Consider the indirect owner-relationship between pod and workload, such as 
pod -> replicaset -> deployment

### CODE REVIEW

File: pkg/util/workloads_utils.go [hunk 1]
```diff
@@ -231,11 +232,13 @@ func GetEmptyWorkloadObject(gvk schema.GroupVersionKind) client.Object {
 		return nil
 	}
 
-	switch gvk.Kind {
-	case ControllerKindDep.Kind:
+	switch gvk.String() {
```
Comment (FillZpp) [inline pkg/util/workloads_utils.go:235]: What's wrong with the previous logic that compares GVK type?

Comment (veophi) [reply pkg/util/workloads_utils.go:235]: @FillZpp previous logic just consider `Kind`,  should we just consider `GroupKind`?

Comment (FillZpp) [reply pkg/util/workloads_utils.go:235]: So how about just compare the GVK type instead of converting to string?

**Review by furykerry — COMMENTED** [review]
/lgtm

**Pull request merged**